cpp/src/slice2swift/Gen.cpp

```diff
@@ -8,6 +8,7 @@
 #include "Ice/StringUtil.h"
 
 #include <algorithm>
+#include <any>
```

Comment:
std::any_of is in `<algorithm>`. `<any>` is something else:
https://en.cppreference.com/w/cpp/header/any.html

Reply:
Hm... I think vscode added this! Weird.